test/rsspp_parser.cpp

```diff
@@ -249,10 +249,11 @@ TEST_CASE("Extracts data from media:... tags in atom feed", "[rsspp::Parser]")
 	REQUIRE(f.items[2].link == "http://example.com/player.html");
 	REQUIRE(f.items[2].author == "John Doe");
 
-	REQUIRE(f.items[3].title ==
-		"using multiple media tags nested in group/content");
```

Comment:
Why did we stop testing the title?

Reply:
See reply to comment on line 288

Reply:
I have undone the change during a rebase